### ISSUE TITLE
[InlineNotification] An improper svg property causes warning in the component.

### ISSUE THREAD

**Issue by miniwangdali** [IBM/carbon-components-react#1593]
## Detailed description
There is a warning said 
```
Warning: Invalid DOM property `fill-rule`. Did you mean `fillRule`?
```

> Is this issue related to a specific component?
It is related to InlineNotification

> What did you expect to happen? What happened instead? What would you like to see changed?
In dev env, I hope there is no warning in the browser console related to the components. But there is it. Although it does not influence the production, it's better to see no warnings...

> What browser are you working in?
Chrome.

> What version of the Carbon Design System are you using?
v6.55.1

## Additional information
The codes which cause the problem might be here:
https://github.com/IBM/carbon-components-react/blob/master/src/components/Notification/a11yIconWarningSolid.js#L28

**Comment by IOIO72** [comment]
Confirming. I'm getting two warnings concerning the warning icon SVG:
https://codesandbox.io/embed/wqqz38o43k

**Comment by emyarod** [comment]
@IOIO72 this should be resolved in https://github.com/IBM/carbon-components-react/pull/1629. Can you verify that the warnings no longer here at this staging link? https://deploy-preview-1629--carbon-components-react.netlify.com/

**Comment by IOIO72** [comment]
> @IOIO72 this should be resolved in #1629. Can you verify that the warnings no longer here at this staging link? https://deploy-preview-1629--carbon-components-react.netlify.com/

Sure! It works fine now. Thank you! 👍
